tox.ini

```diff
@@ -23,12 +22,16 @@ changedir =
 deps =
     notebooks: astroquery
     notebooks: pyyaml
-    devdeps: git+https://github.com/glue-viz/glue
+    dev: git+https://github.com/glue-viz/glue
 extras =
     test: test
+    all: all
```

Comment:
What's the `all` group supposed to comprise that you're adding here? I don't see any differences in packages installed or tests run between `py311-test` and `py311-test-all`.